### ISSUE TITLE
Signature image not displayed in email notifications (Gmail)

### ISSUE THREAD

**Issue by mikejpeters** [verbb/formie#985]
### Describe the bug

When a form has a signature field and email notifications set up that includes the signature field, the image isn't rendered in Gmail. I believe this is because base64 encoded images (and perhaps any directly encoded images) are not supported by Gmail. Probably this is an issue in other email clients as well.

### Steps to reproduce

1. Create a form with a signature field
2. Add an Email Notification for the form with a recipient Gmail address and include _All Form Fields_ in the Email Content
3. Make a form submission, making sure to enter a signature
4. View resulting email in Gmail (signature will not be visible)

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

### Craft CMS version

3.7.48

### Plugin version

1.6.3

### Multi-site?

No

**Comment by engram-design** [comment]
Yeah, it's a known issue with some email clients using `base64` images, as you've discovered. I have some ideas, such as providing a controller action to proxy the request to resolve the signature. So the email notification template for the signature field might look like: `{{ value.getRemoteUrl() }}` - where `value` is a new SignatureImage model.

This would result in something like: `https://yoursite.com/actions/formie/signature?hash=xxxxxxxxxxxxxx` that would return the image. That way, email clients won't complain, and we don't need to create literal image files for email signatures.

**Comment by RitterKnightCreative** [comment]
Using office 365 but getting a similar thing. However what's interesting is if I set the Email Content in the notification to use All Form Fields it'll display just fine but if I single it out, it just spits out the base64 encoded version eg:

![Screen Shot 2022-07-24 at 1 57 39 PM](https://user-images.githubusercontent.com/3707869/180661886-526b12ca-9275-4936-a6d1-4ef55911d0d2.png)

This ends up getting rendered like:

![Screen Shot 2022-07-24 at 1 54 42 PM](https://user-images.githubusercontent.com/3707869/180661899-b2de777e-1db3-4444-9191-438d5dcc021d.png)

As long as it renders fine in a PDF attachment that works for me.

**Comment by engram-design** [comment]
Fixed/improved for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.10](https://github.com/verbb/formie/releases/tag/2.0.10)
